tests/data_files/Readme-x509.txt

```diff
@@ -91,3 +95,13 @@ Signing CA in parentheses (same meaning as certificates).
 
 Note: crl_future would revoke server9 and cert_sha384.crt if signed by CA 1
       crl-rsa-pss* would revoke server6.crt if signed by CA 2
+
+Generation
+----------
+
+Newer test files have been generated through commands in the Makefile. The
+resulting files are committed to the repository so that the tests can
+run without having to re-do the generation and so that the output is the
+same for everyone (the generation process is randomized).
```

Comment:
A makefile is a convenient way to record the origin. It does not add any extra burden compared with writing the same documentation in English and it has many advantages: it can be understood by a computer as well, and it guarantees that the record is accurate with respect to the iinitial generation. Ideally I suppose we'd only keep the makefile and the source files plus a PRNG seed, but that's a lot of infrastructure to add.

Why cant we just keep the files you generated and record their origin? Well, that's how it was before, except that by and large people didn't bother to record the origin.